config/service_providers.yml

```diff
@@ -270,3 +270,20 @@ production:
     cert: 'identity_dashboard_cert'
     attribute_bundle:
       - email
+
+  # DHS / CBP / HRM
+  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:DHS:CBP-HRM-DEV':
+    acs_url: 'https://uxvnwg001a5418.sat.cbp.dhs.gov:9002/hrm-services/saml/SSO'
+    assertion_consumer_logout_service_url: 'https://uxvnwg001a5418.sat.cbp.dhs.gov:9002/hrm-services/saml/logout'
+    sp_initiated_login_url: 'https://uxvnwg001a5418.sat.cbp.dhs.gov:9002/hrm-services/saml/login'
```

Comment:
is this an actual route in the app? I know in the case if micropurchase I added a `/login` route here because that's what other apps had but it wasn't actually defined in the app....